lib/kinetic_cafe/error_dsl.rb

```diff
@@ -115,7 +118,7 @@ def define_error(options)
       i18n_key = "#{i18n_key_base}.#{key}"
 
       if const_defined?(error_name)
-        message = "key:#{key} already exists as #{error_name}"
+        message = String.new("key:#{key} already exists as #{error_name}")
```

Comment:
At the very top we are using `frozen_string_literal: true`, so the strings in this file are frozen by default. Using `String.new(string)` creates a mutable string.